### ISSUE TITLE
Record interview with Linda for info. to create checklist for conducting rapid qualitative data analysis on interview data

### ISSUE THREAD

**Issue by dialectic51** [hackforla/internship#330]
### Dependencies
Linda is currently busy and needs to prepare before she would be ready for an interview.

### Overview
We need to conduct this interview with Linda so that we have a firm understanding of the method of rapid qualitative analysis and can construct a guide or checklist for the method so that it can be used by the Internship group and other projects at Hack4LA.

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, etc.

- [ ] Schedule interview/recording session date with Linda.
- [ ] Transcribe interview using Otter.ai.
- [ ] Collect additional research on rapid qualitative data analysis.
- [ ] Write a checklist for rapid qualitative analysis using template in resources section.
- [ ] Create google file folder and identify appropriate file location on shared google drive.
- [ ] Save checklist as a google document to the shared google drive.

### Resources/Instructions

- [Research Resources on the Internship Wiki](https://github.com/hackforla/internship/wiki/Research) 
- [TWE Report Template to use for writing up findings](https://docs.google.com/document/d/16GP2QU_41dOBBadLUxEurdHh3FjzbkRMKTScr66pSIQ/template/preview)

**Comment by bradyse** [comment]
### Information to Gather from Linda
1. What is the minimum information that needs to be included in the interview guide?
1. Should the interview guide be created in a separate document as the research plan or should it be included in the same research plan document?
1. Before the analysis is conducted, what are the requirements or dependencies that need to happen first? General thoughts below:
    - A research plan/interview guide has been created
    - Transcripts are available
    - Domains have been determined
1. What are the assets that we need to create? (is it only the spreadsheet?)
1. How do you determine the number of coders per interview transcripts and domains?
1. Assuming the interview has already be conducted and transcripts have been prepared, what are the primary tasks of the lead data analyst?
1. Are there any "nuts and bolts" details that need to happen in a specific order of priority?

**Comment by bradyse** [comment]
@dialectic51 

I made some updates in this issue. I also added @lkawentel 's presentation to TWE. Linda, I rebranded your talk to match TWE's design process. Please let me know if you have any questions. Remember that the ultimate goal is for us to create some issue templates so that we can start analyzing the massive amounts of data that we have on this project. 😅

**Comment by dialectic51** [comment]
There are a variety of free Computer Assisted Qualitative Data AnalysiS (CAQDAS) programs, but many have limitations (such as no longer being updated or possibly difficult to use on mac's). Excel may be our best bet. We should investigate what other Hack4LA projects have used to code their qualitative data.

Lists of free qualitative data analysis software:

- [Sources from Illinois Library](https://guides.library.illinois.edu/c.php?g=348074&p=2346107)
- [Sources from Sheridan College](https://sheridancollege.libguides.com/c.php?g=704010&p=5179994)
- [QDA Miner Lite](https://provalisresearch.com/products/qualitative-data-analysis-software/freeware/) may be the best alternative to Excel, but may be tricky to use on mac's.

How to code qualitative data using Excel:
- [Using Microsoft Excel to code and thematically analyse qualitative data: a simple, cost-effective approach--downloads pdf file of journal article](https://ojs.aishe.org/index.php/aishe-j/article/download/281/467/1366)
- [Using Excel and Word to Structure Qualitative Data--journal article](https://www.researchgate.net/publication/306323765_Using_Excel_and_Word_to_Structure_Qualitative_Data)
- [Using Excel for Qualitative Data Analysis])https://web.archive.org/web/20150312021448/http://www.qualitative-researcher.com/qualitative-analysis/using-excel-for-qualitative-data-analysis/)
- [How to Analyze Qualitative Data in Excel (with Easy Steps)](https://www.exceldemy.com/analyze-qualitative-data-in-excel/)

**Comment by lkawentel** [comment]
My responses to the questions are below. 

**1.	What is the minimum information that needs to be included in the interview guide?**
I would recommend the interview guide contain an introductory script, your interview questions, probes (if conducting a semi-structured interview), and a closing script. 
The introductory script should include things like: an introduction to yourself (e.g., your name, role of the project), the purpose of the study, how the data will be used/what products will result from the study, how you will protect participant confidentiality, consent to participate and record (assuming you are recording).

**2.	Should the interview guide be created in a separate document as the research plan or should it be included in the same research plan document?**
This is up to the researcher and/or the researcher’s organization. I like to have my interview guide as an appendix to my research/evaluation plan. 
 
**3.	Before the analysis is conducted, what are the requirements or dependencies that need to happen first? General thoughts below:
o	A research plan/interview guide has been created
o	Transcripts are available
o	Domains have been determined**

Before analysis can begin you will obviously need collected data. You can’t start analysis until you have collected at least some data. In addition to data, I would recommend having an analysis plan, including specifics such as:
- the type of analysis that will be conducted, e.g., thematic analysis, content analysis, narrative analysis, etc.
- (if coding) whether codes will be developed a priori (e.g., directed content analysis) or strictly from a review of the data (e.g., conventional content analysis)
- the software/tools that will be used to conduct the analysis (e.g., NVivo, MAXQDA, Atlas.TI, Dedoose, RQDA, Excel, etc.)
- the lead analyst and team that will conduct the analysis 
- estimated timeline for conducting the analysis

Also, I don't view transcripts as a dependency to qualitative analysis. Same with domains. While I prefer to use transcripts, some qualitative researchers prefer to conduct their analysis from audio recordings, thus bypassing transcription. As for domains, they are an important step in Alison Hamilton’s rapid qualitative analysis process but are not necessary if using other qualitative approaches.

**4.	What are the assets that we need to create? (is it only the spreadsheet?)**
I think this question is asking about rapid qualitative analysis. If so, summary templates (which can be done in Word) and a matrix spreadsheet (which can be done in Excel) are important. Once the matrix is completed, the researcher/team may wish to create additional documents to facilitate analysis, e.g., a codebook in Word or Excel.
  
**5.	How do you determine the number of coders per interview transcripts and domains?**
This depends on a number of factors, e.g., the number of transcripts, the length of the transcripts, how complex the interview data is, what your timeline for completion is, how many hours your analysts can work, whether your team is experienced in qualitative analysis, etc. 

**6.	Assuming the interview has already be conducted and transcripts have been prepared, what are the primary tasks of the lead data analyst?**
The lead analyst should guide the team in conducting the analysis. This can involve a number of tasks, e.g., helping develop the analysis plan, assembling the analysis team, coaching the team on the analytical approach being used (e.g., thematic analysis), assigning tasks to the other analysts, reviewing/checking junior analysts’ work for consistency, and showing junior analysts how to code, summarize, write memos, etc.

**7.	Are there any "nuts and bolts" details that need to happen in a specific order of priority?**
I would recommend consulting an established qualitative checklist to answer this question. Two that come to mind are Western Michigan University’s Qualitative Evaluation Checklist (see https://wmich.edu/sites/default/files/attachments/u350/2014/qualitativeevalchecklist.pdf) and the COREQ (https://academic.oup.com/intqhc/article/19/6/349/1791966). Both checklists are open-access.

**Comment by dialectic51** [comment]
@bradyse Regarding the dependency at the top, I'm going to go ahead and create a new issue under internship (instead of UI/UX). This is because I will be creating a checklist of action items for conducting Rapid Qualitative Analysis (RQA)--not the actual Guide for conducting RQA, which would more appropriately be placed in UI/UX.

**Comment by bradyse** [comment]
@dialectic51 Bonnie said that you can create the issue to create the guide in the TWE repository but brand it according to the standard Hack for LA guides that will be shared in the UI/UX CoP. Bonnie tracked this issue in the guides tracker board.
